index.js

```diff
@@ -78,6 +78,10 @@ GameEngine.prototype.createGameFromMap = function (mapFilePath){
                 game.addHealthWell(j,k);
             } else if (map[j][k] === 'IM'){
                 game.addImpassable(j,k);
+            } else if (map[j][k] === 'S1') {

```

Comment:
After looking at the rest of the code, I think it's safe to do a string match here for matching S1/S2/SP. It's fine if it matches S + any digit so that battles containing more than 2 teams can battle (eventually).